api/nakadi-event-bus-api.yaml

```diff
@@ -116,28 +116,6 @@ securityDefinitions:
         Grants access for consuming Event streams.
 
 paths:
-  /metrics:
```

Comment:
(to be precise, the resource was there but there was no content being returned, ie it didn't have any information.)